src/main/java/edu/harvard/iq/dataverse/api/exposedsettings/Setting.java

```diff
@@ -0,0 +1,38 @@
+package edu.harvard.iq.dataverse.api.exposedsettings;
+
+public class Setting<T> extends SettingItem {
+    /**
+     * Setting names for group {@link edu.harvard.iq.dataverse.api.exposedsettings.SettingGroup#GROUP_NAME_DATAVERSE}
+     */
+    public static final String SETTING_NAME_FQDN = "fqdn";
+    public static final String SETTING_NAME_IS_PUBLIC_INSTALL = "isPublicInstall";
+
+    /**
+     * Setting names for group {@link edu.harvard.iq.dataverse.api.exposedsettings.SettingGroup#GROUP_NAME_API}
+     */
+    public static final String SETTING_NAME_API_TERMS_OF_USE = "termsOfUse";
+    public static final String SETTING_NAME_API_ALLOW_INCOMPLETE_METADATA = "allowIncompleteMetadata";
+
+    /**
+     * Setting names for group {@link edu.harvard.iq.dataverse.api.exposedsettings.SettingGroup#GROUP_NAME_DATASET}
+     */
+    public static final String SETTING_NAME_DATASET_ZIP_DOWNLOAD_LIMIT = "zipDownloadLimit";
+    public static final String SETTING_NAME_DATASET_PUBLISH_POPUP_CUSTOM_TEXT = "publishPopupCustomText";
+    public static final String SETTING_NAME_DATASET_ALLOWED_CURATION_LABELS = "allowedCurationLabels";
+
+    /**
+     * Setting names for group {@link edu.harvard.iq.dataverse.api.exposedsettings.SettingGroup#GROUP_NAME_DATAFILE}
+     */
+    public static final String SETTING_NAME_DATAFILE_MAX_EMBARGO_DURATION_IN_MONTHS = "maxEmbargoDurationInMonths";
```

Comment:
I guess I was sort of hoping there would be a bit more structure in the Java code itself, like in JvmSettings.java.

Here's an example from `dataverse.auth.oidc.enabled=1`:

```
    PREFIX("dataverse"),
    SCOPE_AUTH(PREFIX, "auth"),
    SCOPE_OIDC(SCOPE_AUTH, "oidc"),
    OIDC_ENABLED(SCOPE_OIDC, "enabled"),
```

That said, what's in the PR is fine.

Reply:
The structure here is datafile/maxEmbargoDurationInMonths.

Being "datafile" the subgroup and "maxEmbargoDurationInMonths" the setting to access. It's the only embargo-related setting I've seen, which is why I haven't created a subgroup "embargo" under "datafile". I don't think more embargo settings will be created in the near future, that's why I didn't create the subgroup.

Can you think of a better way to group it?